README.adoc

```diff
@@ -104,3 +104,99 @@ include::topics/templates/document-attributes.adoc[]
 . Open the generated html file to ensure everything looks correct.
 
 You have now created the scaffolding for a new guide. When you are ready for some initial review and feedback, file a pull request to `upstream/master`.
+
+### Merging Contributions
+// title OK?
+// following content up for revision.
+// add note on branching according to issue numbers?
```

Comment:
IMO it's a nice to have, but not critical as we'll have all sorts of branch names anyway from people who are not writers.